GoogleChromeLabs/ps-analysis-tool
# Fix: Clear cookie preview area

## Description
This PR aims to fix the cookie preview area once the cookie is not present anymore in the table view.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch.
- In the terminal run `npm start`.
- Open a fresh instance of the browser and analyse the page with the PSAT tool in DevTool.
- In the table select a 1P cookie then apply the `scope` filter for 3P cookies, you should see the cookie details area being cleared.


## Screenshot/Screencast

https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/59614577/774710f7-1f47-4150-88c8-26a6f6ccfd62


---

## Checklist





Fixes #351
